yb-voyager/src/migassessment/common.go

```diff
@@ -36,7 +36,6 @@ type SizingRecommendation struct {
 	OptimalSelectConnectionsPerNode int64
 	OptimalInsertConnectionsPerNode int64
 	EstimatedTimeInMinForImport     float64
-	ParallelVoyagerJobs             float64
```

Comment:
General question: We're removing parallel-jobs recommendation right now, but we are not changing the `EstimatedTimeInMinForImport` based on new experiments with adaptive-parallelism. 

Is that a conscious decision?